drivers/OSPI.h

```diff
@@ -0,0 +1,272 @@
+/* mbed Microcontroller Library
```

Comment:
This is fine for now but in the future I would like to see a file which would have the parts shared by SPI, QSPI and OSPI.